components/select/docs/partials/api.md

```diff
@@ -247,28 +247,6 @@ This example programmatically adds the `error` state when a user selects an opti
 
 </auro-accordion>
 
-### Watch for value changes
-
-The following example listens for the `selectedOption` event. Once triggered, `element.value` or `element.optionSelected` may be queried for the new value or complete option object.
-
-<div class="exampleWrapper">
-  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../apiExamples/valueAlert.html) -->
-  <!-- AURO-GENERATED-CONTENT:END -->
-</div>
-
-<auro-accordion alignRight>
-  <span slot="trigger">See code</span>
-
-
-<!-- AURO-GENERATED-CONTENT:START (CODE:src=./alertValue.js) -->
-<!-- AURO-GENERATED-CONTENT:END -->
-
-<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../apiExamples/valueAlert.html) -->
-<!-- AURO-GENERATED-CONTENT:END -->
-
-</auro-accordion>
-
-
```

Comment:
Shouldn't this example just have been updated to use the correct event rather than the deprecated event?

Reply:
The reason that this example was deleted is because it was never working in the first place. It has been broken and the reason is is because it was calling an event that never existed in the component. It kept calling `this.addEventListener('selectedOption')` when that event was actually being dispatched from the menu level.

Alternatively, I can reinstate this example and change it to use the menu event in the example code.